src/sdg/core/management/utils.py

```diff
@@ -165,6 +168,14 @@ def load_upn(data: List[Dict[str, Any]]) -> int:
         if created:
             count += 1
 
+    all_upns = UniformeProductnaam.objects.all()
+
+    deleted_upns = [x for x in all_upns if x not in upn_updated_list]
+
+    for upn in deleted_upns:
+        upn.is_verwijderd = True
+        upn.save()
```

Comment:
These kind of for-loops are dangerous if the whol function is not done in a transaction (which doesn't need to be the case here). The `upn` is the whole instance, and you are saving the whole instance. If it was changed (by a user) between line 171 and 177, the wrong information is saved. You can solve this in 2 ways:

1. Use fields: `upn.save(updated_fields=["is_verwijderd", ])` -- See: https://docs.djangoproject.com/en/4.1/ref/models/instances/#ref-models-update-fields
2. Use a bulk-update: `deleted_upns` is not a queryset, so we cannot do `.update(...)` on it. We can however, make it into a queryset (also replace 7 lines of code into 1):

```suggestion
    UniformeProductnaam.objects.filter(pk__not_in=[upn.pk for upn in upn_updated_list]).update(is_verwijderd=False)
```

Can even be nicer if you just store the PK in the `upn_updated_list` directly, instead of the object. This also saves memory!